health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/BoundaryRelationCreate.js

```
@@ -0,0 +1,682 @@
+        setNewBoundaryData((prevItems) => {
+            const filteredData = prevItems.filter((item, idx) => idx !== index);
+            return filteredData;
+        })
+    }
+
+    const callCreate = async () => {
+        let defData = boundaryData;
+        if (newHierarchy === true) {
+            defData = [];
+        }
+        let flag = false;
```

Biome

[error] This let declares a variable that is only assigned once.

'flag' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)